server/routes/api.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+
+const gainzController = require('../controllers/gainzController');
+
+const router = express.Router();
+
+router.get('/',
+  gainzController.getExcercises,
+  (req, res) => res.status(200).json(res.locals.excerciseQuery),
+);
+
+router.get('/history',
+  gainzController.getHistory,
+  (req, res) => res.status(200).json(res.locals.drillQuery),
+);
+
+// router.get('/createExercise',
+//   gainzController.getNewExcercises,
+//   (req, res) => res.status(200).json(res.locals.homeQuery),
+// );
+
+// router.get('/login',
+//   gainzController.getLogin,
+//   (req, res) => res.status(200).json({}),
+// );
+
+// router.get('/logout',
+//   gainzController.getFilm,
```

Comment:
Why is getFilm here? Please don't copy-paste. You will grow as a developer if you can type this yourself from memory!